tasks/create_record.json

```diff
@@ -0,0 +1,17 @@
+{
+    "puppet_task_version": 1,
+    "supports_noop": false,
+    "description": "Create a ServiceNow record",
+    "files": ["ruby_task_helper/files/task_helper.rb"],
+    "remote": true,
+    "parameters": {
+        "table": {
+            "description": "ServiceNow table",
+            "type": "String[1]"
+        },
+        "servicenow_fields": {
+            "description": "ServiceNow fields",
+            "type": "Hash[String, String]"
```

Comment:
Hmm, I think we'll want the value-type to be Puppet's equivalent of `Any Type` instead of `String` since ServiceNow fields can have arbitrary types (String, Integer, Boolean). Otherwise, if I have a numeric field `A` and I pass in `A=3` to the task, the API call will fail b/c the endpoint will receive it as `"A": "3"` instead of `"A": 3`.

Reply:
fixed.